zebrad/src/commands/generate.rs

```diff
@@ -19,23 +19,23 @@ impl Runnable for GenerateCmd {
 #
 # This file can be used as a skeleton for custom configs.
 #
-# This file is generated using zebrad's current defaults. If you want zebrad
-# to automatically use any newer defaults, set the config options you want to
-# keep, and delete the rest.
+# Unspecified fields use default values. Optional fields are Some(field) if the
+# field is present and None if it is absent.
 #
-# The config format is documented here:
+# This file is generated as an example using zebrad's current defaults.
+# You should set only the config options you want to keep, and delete the rest.
+# Only a subset of fields are present in the skeleton, since optional values
+# whose default is None are omitted.
+#
+# The config format (including a complete list of sections and fields) is
+# documented here:
```

Comment:
not including this means people might never realize that there are missing fields not included in the skeleton